### ISSUE TITLE
Missing API to set aria-label to labelless TextField

### ISSUE THREAD

**Issue by OlliTietavainenVaadin** [vaadin/vaadin-text-field#454]
In the case of a TextField without a label, there should still be a way to set the `aria-label` attribute for accessibility reasons. The internal implementation `input` element is not available from the component so it's not possible to add the attribute manually.

**Comment by alvarezguille** [comment]
Adding enhancement label as this use case was not considered in initial design.

**Comment by rolfsmeds** [comment]
`aria-labelledby` should be removed when no label is set: https://github.com/vaadin/vaadin-text-field/issues/562
